src/modules/users/components/AvatarDropdown/AvatarDropdownPopoverMobile.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Button from '~core/Button';
+import { Colony } from '~data/index';
+import DropdownMenu, {
+  DropdownMenuSection,
+  DropdownMenuItem,
+} from '~core/DropdownMenu';
+import MaskedAddress from '~core/MaskedAddress';
+import MemberReputation from '~core/MemberReputation';
+import UserTokenActivationDisplay from '../UserTokenActivationButton/UserTokenActivationDisplay';
```

Comment:
Aren't the `~core/abc` syntax and `../xyz` syntax both examples of relative imports?

Reply:
Or `~core` is absolute? And therefore should be below the `../`?

Reply:
Core should always be first when dealing with component imports.

Here's a rule of thumb for import priorities:
1. core
2. user
3. dashboard
4 (last). local imports (same folder as current component)

_(page should be before local, but you'll rarely have to deal with that)_

Reply:
Yeah, my bad for the misunderstanding but it's as Raul said.  We normally order these imports based on location and if they are relative or absolute, so in this case the import of `UserTokenActivationDisplay` is quite a local import compared to the imports of `~core` and `~types` that you have here, and the more local an import is, the more it should stay below

For this case, I would put this import between the import of `TokenActivationPopover` (or just above,  they have the same level of "localness" after all) and `AppState` because it's "as local as" the import `TokenActivationPopover` but not too direct like `AppState`, which is imported from the same folder.

Reply:
I've grouped the `../` and put them below the `~xx` as I have seen this elsewhere in the codebase.

Reply:
Ok thanks both for clarifying. Wrote the above before I saw your comments but I got the jist of it anyhow